### DISCUSSION TITLE
Installation Bitcraze/Toolbelt in Docker  (Windows)  - Create Bash-Alias?

### DISCUSSION THREAD

**Discussion by Drondelbee** [bitcraze/discussions#755] [Q&A]
I cant execute make command. I forgot to install Bitcraze/Toolbelt and Docker App.

How I add a alias to my ./bashrc oder ./profile analog to Installation guide?

https://hub.docker.com/r/bitcraze/toolbelt

![Docker Screen](https://github.com/bitcraze/discussions/assets/74738815/15606b7b-1a3f-4171-8d3a-8f56a5d520ef)

**Comment by knmcguire** [comment]
Hi!

Some questions about your problem first:
- Which Windows are you running: Windows 10 or Windows 11 ?
- Do you have WSL2 configured on your Windows?

**Reply by Drondelbee** [reply]
Hi,

- Windows 10
 

- `wsl -v`
WSL-Version: 1.2.5.0
Kernelversion: 5.15.90.1
WSLg-Version: 1.0.51

- `wsl --list --verbose`
  NAME                           STATE           VERSION
* Ubuntu                         Running             2
  docker-desktop             Running            2
  docker-desktop-data     Running            2
- `wsl -v`

 -v
WSL
Wsman Shell commandLine, version 0.2.1

USAGE: wsl COMMAND [PARAMS...]

COMMANDS:
identify  - WS-Identify
enum      - WS-Enumerate
get       - WS-Get
put       - WS-Put
invoke    - WS-Invoke
xclean    - Delete all files generated by this tool set
xcred     - Create or display credential file
xcert     - Get server certificate (saved to <IPADDRESS>.crt)

PARAMS specification is specific to a COMMAND.

Output will be saved to ./response.xml. If you want to run parallel
executions in the same directory, define RTFILEPREFIX in the environment.
Doing so may significantly increase files generated.

Requires: curl, xmllint, GNU core utilities.
Optional: xsltproc for output formatting, gpg for encrypted credential.
Optional: wget as alternate for curl when not available.

**Reply by knmcguire** [reply]
Alright this seems to be WSL2, but you probably don't have gui support as that mostly only works on windows 11 I believe.

Are you able to open visual studio code remotely for your WSL? You can check this by opening a WSL terminal and type 'code'
